source/agora/node/Registry.d

```diff
@@ -351,8 +351,8 @@ public class NameRegistry: NameRegistryAPI
     {
         Message reply;
         reply.header.RCODE = Header.RCode.FormatError;
-        reply.header.RA = false; // TODO: Implement
-        reply.header.AA = true;  // TODO: Make configurable
+        reply.header.RA = false;
+        reply.header.AA = true;
```

Comment:
Caching shouldn't be AA should it ?

Reply:
[This](https://github.com/bosagora/agora/pull/2703/files#diff-9a7d511b8407367630fb104c60937f8c304fb9d6eed6ff2281d46541c93513f9R1174) handles that. I will move setting those fields into `answer` method withing the zone.